.github/workflows/conformance-aks.yaml

```diff
@@ -191,7 +191,7 @@ jobs:
 
       - name: Run connectivity test
         run: |
-          cilium connectivity test --flow-validation=disabled
+          cilium connectivity test --flow-validation=warning
```

Comment:
@nebril Was this merged by mistake? I had understood it was added to debug a flake in this PR, but wasn't meant to be merged. It's effectively reverting https://github.com/cilium/cilium/pull/16388 which was aimed to reduce flakes until flow validation works better.

Reply:
I have the same question :)

Reply:
Especially since the commit does not change `flow-validation` in all instances, some of them are still disabled 🤔

Reply:
Discussed offline: am reverting to a consistent state with flow validation `disabled` as per #16388, we can always switch back to any other level once we are confident the CLI has improved.